eventmesh-connectors/eventmesh-connector-rocketmq/src/main/java/org/apache/eventmesh/connector/rocketmq/source/connector/RocketMQSourceConnector.java

```diff
@@ -220,7 +220,7 @@ public List<ConnectRecord> poll() {
             byte[] body = messageExt.getBody();
             String bodyStr = new String(body, StandardCharsets.UTF_8);
             RecordPartition recordPartition = convertToRecordPartition(messageExt.getTopic(),
-                messageExt.getBrokerName(), messageExt.getQueueId());
+                    messageExt.getBrokerName(), messageExt.getQueueId());
             RecordOffset recordOffset = convertToRecordOffset(messageExt.getQueueOffset());
```

Comment:
There is still additional indentation here. Is it because the rules didn't cover it?

Reply:
Yes, this is because I excluded this file from the spotless conversion list first, and fixed the indentation issue in a later commit.
I'll fix this in a new round of commits.